src/main/java/org/betonquest/betonquest/menu/Slots.java

```diff
@@ -216,7 +219,7 @@ public enum Type {
         RECTANGLE
     }
 
-    public static class SlotException extends Exception {
+    public static class SlotException extends QuestException {
```

Comment:
This Exception should now prabably be deleted and the QuestException should be used directly